src/util/amount.rs

```diff
@@ -995,6 +995,184 @@ pub mod serde {
     }
 }
 
+/// Type for dealing with unknown amounts
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[derive(Debug, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Both variants are simple enough so let's give them some flexibility :wink:

Reply:
Eq, Hash cannot be implemented on f64.

I did add Clone, Copy, and PartialOrd.